bio/mofaflex/training/wrapper.py

```
@@ -0,0 +1,46 @@
+        mdata = md.read_h5mu(mdata_path)
+    elif mdata_path.suffix == ".h5ad":
+        mdata = md.read_h5ad(mdata_path)
+    else:
+        raise Exception("Illegal file format. Must be h5mu or h5ad.")
+
+    model: mfl.MOFAFLEX = train_model(
+        mdata=mdata,
+        out_path=Path(snakemake.output[0]),
+        n_factors=snakemake.params.get("n_factors", 15),
+        likelihoods=snakemake.params.get("likelihoods", "Normal"),
+        batch_size=nakemake.params.get("batch_size", 1000),
```

Ruff (0.15.18)

[error] Undefined name `nakemake`

(F821)